docs/layout-props.md

```diff
@@ -167,7 +167,7 @@ const Square = () => (
 
 const randomHexColor = () => {
   return '#000000'.replace(/0/g, () => {
-    return (~~(Math.random() * 16)).toString(16);
+    return Math.round(Math.random() * 16).toString(16);
```

Comment:
This was previously doing two's complement negation twice, but we can also just round for a similar effect (tested this out). The ESLint config bans bitwise operators.

Reply:
TIL!